e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_id_attrib_spec.ts

```diff
@@ -72,13 +72,13 @@ describe('AD / LDAP', () => {
         const randomId = getRandomId();
         const newFirstName = `Firstname${randomId}`;
         const newLastName = `Lastname${randomId}`;
-        cy.updateLDAPUser({
+        (cy as any).updateLDAPUser({
```

Comment:
this is not an issue once you update to latest master

Reply:
updated